besser/bot/library/event/event_library.py

```diff
@@ -59,3 +59,24 @@ def variable_matches_operation(session: 'Session', event_params: dict) -> bool:
     operation: Callable[[Any, Any], bool] = event_params['operation']
     current_value: Any = session.get(var_name)
     return operation(current_value, target_value)
+
+
+def file_received(session: 'Session', event_params: dict) -> bool:
+    """
+    This event only returns True if the session variable "file" was set to true.
+
+    Args:
+        session (Session): the current user session
+        event_params (dict): the event parameters
+
+    Returns:
+        bool: always false
+    """
+    if session.get("file_received"):
```

Comment:
Something general: since we are using some session variables as "internal" ones (e.g. file, file_received, also in the variable_matches_operation event: var_name, target, operation...) we may think a way to reformat this in a less "dirty" way. For example, having a separate attribute in the session dedicated to these internal variables. This is what I tried to do with session.message, although I don't see having a specific attribute in teh session for each of these variables. Maybe putting them all together in another dict object? To think about it

Reply:
I agree that if feels a bit "dirty" storing random necessary variables in the session. I think that going the way as with "session.message" might be a good idea. I doubt that we will have that many special internal variables that this method could be disadvantageous, as it's a good way to differentiate between chatbot creator variables and framework developer variables